worker/worker-manager/score-worker-manager-impl/src/main/java/io/cloudslang/worker/management/services/InBuffer.java

```diff
@@ -41,38 +41,41 @@
     private static final Logger logger = Logger.getLogger(InBuffer.class);
 
     private final static long MEMORY_THRESHOLD = 50000000; // 50 Mega byte
+    private final static int MINIMUM_GC_DELTA = 10000; // minimum delta between garbage collections in milliseconds
```

Comment:
??? @sekler @natabeck ???
I find it hard to believe you do not want to even give us some flag to turn this func off...
this can cause big problems at users ...